test/e2e/util.go

```diff
@@ -1134,18 +1134,21 @@ func testContainerOutputMatcher(scenarioName string,
 	if err != nil {
 		Failf("Failed to get pod status: %v", err)
 	}
+	testContainerOutputMatcherNoPodInit(scenarioName, c, pod, containerName, podStatus.Spec.NodeName, ns, expectedOutput, matcher)
+}
 
-	By(fmt.Sprintf("Trying to get logs from node %s pod %s container %s: %v",
-		podStatus.Spec.NodeName, podStatus.Name, containerName, err))
+// checkStringsInContainerLogs fetch containerName logs and detect if string in expectOutput are present
+func testContainerOutputMatcherNoPodInit(scenarioName string, c *client.Client, pod *api.Pod, containerName, nodeName string, namespace string, expectedOutput []string,
```

Comment:
scenarioName isn't used here.

also, the name of this function is pretty awkward (and its docstring doesn't match). something like `matchContainerLog` or something is probably better.